gemelli/_rpca_defaults.py

```diff
@@ -0,0 +1,36 @@
+# ----------------------------------------------------------------------------
+# Copyright (c) 2019--, gemelli development team.
+#
+# Distributed under the terms of the Modified BSD License.
+#
+# The full license is in the file COPYING.txt, distributed with this software.
+# ----------------------------------------------------------------------------
+
+# Configuration file where you can set the parameter default values and
+# descriptions. This is used by both the standalone RPCA and QIIME 2 RPCA sides
+# of gemelli.
+
+DEFAULT_RANK = 3
```

Comment:
Are any of these shared with ctf? May make sense to have a single file with defaults if you envision wanting modify  a single default/description that is shared by ctf/rpca

Reply:
These have been merged into one file and reduced.